Class/Functions.ts

```diff
@@ -0,0 +1,18 @@
+export default class Functions {
```

Comment:
export class not default

Reply:
and why a class ? not a function ?

Reply:
I prefere a class, to stay in the POO, and if we add other function in the futur

Reply:
the problem is that a class is less efficient for functions reuse so the use of exported function makes more sense